src/main/repository/UserRepository.ts

```diff
@@ -11,7 +11,7 @@ export class UserRepository extends Repository<User> {
 
   findByFirebaseId(
     firebaseId: string,
-    options: FindOneOptions<User>
+    options?: FindOneOptions<User>
   ): Promise<User | undefined> {
```

Comment:
Oo I realized we can do this!
```suggestion
    options?: FindOneOptions<User>
  ): Promise<User> {
```